test/UnitTests/MSTestAdapter.UnitTests/Execution/ExceptionHelperTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.VisualStudio.TestPlatform.MSTestAdapter.UnitTests.Execution
+{
+    [TestClass]
+    public class ExceptionHelperTests
```

Comment:
We are not using MSTest to test MSTest for some historical reason.

```suggestion
    public sealed class ExceptionHelperTests : TestContainer
```